### PR TITLE
[Docs] Improve Chinese and English build.md

### PR DESCRIPTION
**Pull request by PeterH0323**
## Motivation

Imporve the build.md

## Modification

修改中文版 build.md 部分措辞

## Checklist

- [x] 1. Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] 2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
- [x]  3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
- [x]  4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/build.md [hunk 1]
```diff
@@ -98,15 +99,15 @@ pip install -e .
 **Note**
 
 - 有些依赖项是可选的。运行 `pip install -e .` 将进行最小化依赖安装。 如果需安装其他可选依赖项，请执行`pip install -r requirements/optional.txt`，
-或者 `pip install -e . [optional]`。其中，`[optional]`可以填写`all`, `tests`, `build`, `optional`
+或者 `pip install -e . [flag]`。其中，`[flag]`可以替代为：`all`、`tests`、`build` 或 `optional`。
 
 ### 构建 SDK
 
-读者如果只对模型转换感兴趣，那么可以跳过本章节
+如果您只对模型转换感兴趣，那么可以跳过本章节
```
Comment (AllentDan) [inline docs/zh_cn/build.md:106]: 统一称谓，全部是你或者您

Comment (PeterH0323) [reply docs/zh_cn/build.md:106]: Done，刚刚搜索发现其他文档也有很多 `你` 和 `您` 混用了，因为文档阅读对象是使用者（客户），感觉用 `您` 比较合适点，故我全部改为 `您` 了

**Review by AllentDan — APPROVED** [review]
LGTM

File: docs/zh_cn/build.md [hunk 2]
```diff
@@ -98,15 +99,15 @@ pip install -e .
 **Note**
 
 - 有些依赖项是可选的。运行 `pip install -e .` 将进行最小化依赖安装。 如果需安装其他可选依赖项，请执行`pip install -r requirements/optional.txt`，
-或者 `pip install -e . [optional]`。其中，`[optional]`可以填写`all`, `tests`, `build`, `optional`
+或者 `pip install -e . [flag]`。其中，`[flag]`可以替代为：`all`、`tests`、`build` 或 `optional`。
```
[the remark this answers is not in the document]
Comment (PeterH0323) [reply docs/zh_cn/build.md]: Done

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**